congame-doc/introduction.scrbl

```diff
@@ -1,37 +1,530 @@
+
+Click on the @onscreen{Admin} tab. (If you don’t see this, contact your server admin.)
+
+Under the @italic{Studies} section, click @onscreen{New Study}, and fill out the upload form as
+follows:
+
+@itemlist[
+
+@item{@bold{Name}: Enter @racketvalfont{Age Survey} (any human-friendly title will do).}
+
+@item{@bold{Slug}: Leave this blank.}
+
+@item{@bold{Type}: Set to @racketresultfont{Conscript}.}
+
+@item{@bold{Study ID}: Enter @racketvalfont{simple-survey}. This is the identifier we used in the 
+@racket[defstudy] and @racket[provide] expressions in our @filepath{age-survey.rkt} file.}
+
+@item{@bold{Study Source}: Click the @kbd{Browse} button and locate/select the
+@filepath{age-survey.rkt} file.}
+
+]
+
+@screenshot["intro-upload.png"]
+
+When ready, click @kbd{Create} to upload and create the study.
+
+@margin-note{If you receive errors at this point, double check that the contents of
+@filepath{age-survey.rkt} exactly match the contents of the example file shown above. In particular,
+check that the top line says @code{#lang conscript} and that the @racket[(provide simple-survey)]
+expression is present in the file.}
+
+Alternatively, if too much time has passed since you clicked the @onscreen{New Study} link, you may
+receive a “session expired” error — simply fill out the form and try again more quickly this time.}
+
+@;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@subsection{Creating a study instance}
+
+As soon as the upload is complete, you’ll be taken to a screen listing all the @tech{study
+instances} for the newly-created study:
+
+@screenshot["intro-empty-instancelist.png"]
+
+Click on @onscreen{New Instance}. For now, just fill in a value like @racketvalfont{Instance 1}
+in the @italic{Name} field, and leave the rest of the fields at their default values:
+
+@screenshot["intro-new-instance.png"]
+
+Click @kbd{Create} to create an instance of the study.
+
+You’ll be taken back to the study’s instance list, and this time the new instance will appear in the
+list.
+
+@;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@subsection{Trying it out}
+
+Now that you’ve published your study and created an instance for it, you can go click on the
+@onscreen{Dashboard} link and see the new instance listed there. 
+
+You’ll also see a link titled @onscreen{Enroll} — click it! You will be taken through the study
+exactly as you were when you tried it out on your computer earlier. Go ahead and complete all the
+steps.
+
+@margin-note{As long as the study instance is in “active” status, other people who log into this same study as
+participants will see this same study instance and the @onscreen{Enroll} link on their dashboard,
+allowing them to complete the study as well.}
+
+Once you’re done with the study, @mark{navigate back to the dashboard somehow}. Then click on the
+@onscreen{Admin} → @onscreen{Instance 1} link.
+
+@mark{All the way at the bottom}, you’ll see a section titled @bold{Participants} — and you’ll be
```

Comment:
Why the mark, are you unsure this is correct, or you think it is an anti-feature? Our admin page is definitely an evolved beast and ugly.

Reply:
As I was writing the tutorial I asked myself why the Participants should be at the bottom rather than at the top. There may well be a good reason for this (I can think of a couple), but I can easily imagine a newcomer wondering why they need to wade past all the other stuff to get to the one thing they are most interested in from the beginning. Very minor note; I will probably take it out now that you’ve seen it so readers don’t get confused.

Reply:
Can you add making a list of suggested improvements to the admin page as something to look at during the next round? It is one of those pain points we primarily don't update because we got used to it and don't really know how to improve. I agree that putting participants at the top is better since it's the default reason for coming to the admin.